src/plugins/saved_objects_management/public/management_section/object_view/saved_object_view.tsx

```diff
@@ -117,22 +122,56 @@ export class SavedObjectEdition extends Component<
           type={type}
           onDeleteClick={() => this.delete()}
           viewUrl={service.urlFor(id)}
+          useUpdatedUX={useUpdatedUX}
+          navigationUI={navigationUI}
+          application={application}
         />
-        {notFoundType && (
-          <>
-            <EuiSpacer size="s" />
-            <NotFoundErrors type={notFoundType} />
-          </>
-        )}
-        {canEdit && (
-          <>
-            <EuiSpacer size="s" />
-            <Intro />
-          </>
-        )}
+        {notFoundType &&
+          (useUpdatedUX ? (
+            <HeaderControl
+              controls={[
+                {
+                  renderComponent: <NotFoundErrors type={notFoundType} />,
+                },
+              ]}
+              setMountPoint={application.setAppBottomControls}
+            />
+          ) : (
+            <>
+              <EuiSpacer size="s" />
+              <NotFoundErrors type={notFoundType} />
+            </>
+          ))}
+        {canEdit &&
+          (useUpdatedUX ? (
+            <HeaderControl
+              controls={[
+                {
+                  renderComponent: <Intro />,
+                },
+              ]}
+              setMountPoint={application.setAppBottomControls}
+            />
+          ) : (
+            <>
+              <EuiSpacer size="s" />
+              <Intro />
+            </>
+          ))}
         {object && (
           <>
-            <EuiSpacer size="m" />
+            {useUpdatedUX ? (
+              <EuiTitle>
+                <h3>
```

Comment:
Should this be `h2` with `<EuiTitle size="s">`?

Reply:
That looks more reasonable now, have updated in code and description. Thanks.